resources/windows/wix/customactions.wxs.erb

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="windows-1252"?>
```

Comment:
I still don't see any erb variables in this file. If it's static, it shouldn't have the `erb` ending